common/dds-model.h

```diff
@@ -14,41 +16,50 @@ namespace rs2
     {
 
     public:
-        dds_model(rs2::device dev);
+        dds_model( rs2::device dev );
 
-        void render_dds_config_window(ux_window& window, std::string& error_message);
+        void render_dds_config_window( ux_window & window, std::string & error_message );
 
         void open_dds_tool_window();
 
         void close_window() { ImGui::CloseCurrentPopup(); }
 
-        rsutils::type::eth_config get_eth_config(int curr_or_default);
-
-        void set_eth_config(rsutils::type::eth_config &new_config , std::string& error_message);
-
         bool supports_DDS();
 
 
     private:
-
-        enum priority {
+        enum priority
+        {
             ETH_FIRST,
             USB_FIRST,
             DYNAMIC
         };
 
         rs2::device _device;
-
-        rsutils::type::eth_config _default_config;
-        rsutils::type::eth_config _current_config;
-        rsutils::type::eth_config _changed_config;
+        rs2::eth_config_device _eth_device;
+
+        uint32_t _domain_current, _domain_to_set;
+        rs2_eth_link_priority _link_priority_current, _link_priority_to_set;
+        uint32_t _link_timeout_current, _link_timeout_to_set;
+        bool _dhcp_enabled_current, _dhcp_enabled_to_set;
+        uint8_t _dhcp_timeout_current, _dhcp_timeout_to_set;
+        rsutils::type::ip_address _ip_current, _ip_to_set;
+        rsutils::type::ip_address _netmask_current, _netmask_to_set;
```

Comment:
not sure about the ip_adress namespace name
what about network or something similar, more generic than ip_address?

Reply:
Not sure what you indended here.
I think `ip_address` name is well understood, even when used for netmask. Names like `ip_v4_pattern` are more generic but obscure.
`type` generic namespace might be replaced with `network`.